### PR TITLE
RHD-713 - Updated all quickstart tags to most recent

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

File: _jdg-quickstarts [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 7be4b6dfe68cb0d5b623528af18d0bc39471552f
+Subproject commit db2f24eeb88d165544195f4a0696bb82a6a531b6
```
Comment (paulrobinson) [inline _jdg-quickstarts]: This links to JDG_6.5.1.CR1. You should link to the QS tag that matches the latest GA download available on the site. in this case 6.5.0.GA, which doesn't seem to be tagged in: https://github.com/jboss-developer/jboss-jdg-quickstarts/releases. You'll need to track down what happened there.

File: _eap-quickstarts [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 9c2b50a2b68a9d1f9f4fba22d0799964990add9c
+Subproject commit 71a4719e5f52f25857b51b3a4f2e781db5659f45
```
Comment (paulrobinson) [inline _eap-quickstarts]: Looks good

File: _picketlink-quickstarts [hunk 3]
```diff
@@ -1 +1 @@
-Subproject commit 81554884f82c38226973b18b682e4c10ff957298
+Subproject commit 4e01343a65a9ea0df8bd3e850cee17afa81b3736
```
Comment (paulrobinson) [inline _picketlink-quickstarts]: Looks good

File: _portal-quickstarts [hunk 4]
```diff
@@ -1 +1 @@
-Subproject commit caf178764214cd14a6c93b65308343945f755cd1
+Subproject commit c69f4643a0ab20d6777ebb30a00bffec726d890a
```
Comment (paulrobinson) [inline _portal-quickstarts]: Again, I'm not sure this is right. You'll need to find out which one correlates to latest product release.

File: _jdg-quickstarts (hunk 1 above)
Comment (Dantheman720) [inline _jdg-quickstarts]: I don't see a GA release for this so I created https://github.com/jboss-developer/jboss-jdg-quickstarts/issues/101

File: _eap-quickstarts (hunk 2 above)
Comment (paulrobinson) [inline _eap-quickstarts]: This change needs to stay

File: _picketlink-quickstarts (hunk 3 above)
Comment (paulrobinson) [inline _picketlink-quickstarts]: This change needs to stay

File: _portal-quickstarts [hunk 5]
```diff
@@ -1 +1 @@
-Subproject commit caf178764214cd14a6c93b65308343945f755cd1
+Subproject commit e4cf6bbd6e494f2fe31dad92b0751eeb9b977110
```
Comment (paulrobinson) [inline _portal-quickstarts]: Drop this change.

**Comment by paulrobinson** [comment]
@Dantheman720 let me know when this is ready for review.

**Comment by paulrobinson** [comment]
@Dantheman720 can you finish this off today, so it's not hanging around whilst you are away next week?

**Comment by Dantheman720** [comment]
@paulrobinson Yep!  Will do.  I thought I had it ready, it looks like portal snuck its way in.

**Pull request merged**